docs/stdlib/abstract.rst

```diff
@@ -92,40 +92,40 @@ There are a number of abstract numeric types extending ``anyscalar``:
 
     :index: any anytype
 
-    Abstract base scalar type for
-    :eql:type:`anyint`, :eql:type:`anyfloat`, and :eql:type:`decimal`.
+    Represents an abstract base scalar type for
+    :eql:type:`anyint`, :eql:type:`anyfloat` and :eql:type:`decimal` types.
 
 
 Abstract Range Types
 ====================
 
-There are some types that can be used to construct :ref:`ranges
-<ref_std_range>`. These scalar types are distinguished by the following
-abstract types:
+Like abstract numeric types, :ref:`ranges <ref_std_range>` can also be
+specified.
 
 .. eql:type:: std::anypoint
 
-    :index: any anypoint anyrange
+    :index: any anypoint anyrange point
```

Comment:
Recommended not to add these additional search terms to the range types since a user searching for those new terms is not likely to be looking for these abstract types. Same recommendation on line 123.

Reply:
```suggestion
    :index: any anypoint anyrange
```